tools/src/process-allure-reports.py

```diff
@@ -0,0 +1,101 @@
+import os
+import subprocess
+import argparse
+from allure_combine import combine_allure
+
+COMBINE_DIR = 'combine'
+COMPLETE_FILE_NAME = 'index.html'
+PUT_TIMEOUT = 600
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Process allure reports')
+    parser.add_argument('--neofs_node', required=True, type=str, help='NeoFS node')
+    parser.add_argument('--wallet', required=True, type=str, help='Path to the wallet')
+    parser.add_argument('--cid', required=True, type=str, help='Container ID')
+    parser.add_argument('--run_id', required=True, type=str, help='GitHub run ID')
+    parser.add_argument('--allure_report', type=str, help='Path to generated allure report directory',
+                        default='allure_report')
+    return parser.parse_args()
+
+
+def put_combine_result_as_static_page(directory: str, neofs_node: str, wallet: str, cid: str, run_id: str,
+                                      password: str) -> None:
+    base_cmd = (
+        f'NEOFS_CLI_PASSWORD={password} neofs-cli --rpc-endpoint st1.{neofs_node}:8080 '
+        f'--wallet {wallet}  object put --cid {cid} --timeout {PUT_TIMEOUT}s'
+    )
+
+    for subdir, dirs, files in os.walk(directory):
+        current_dir_name = os.path.basename(subdir)
+        for filename in files:
+            filepath = subdir + os.sep + filename
+            base_cmd_with_file = f'{base_cmd} --file {filepath}'
+            if filename == 'complete.html' and current_dir_name == COMBINE_DIR:
+                # allure_combine combines the Allure report and saves it as a static page under the name "complete.html"
+                # Later we will write a patch in allure_combine or fork it, but for now we will rename to "index.html"
+                filename = COMPLETE_FILE_NAME
+                object_cmd = f'{base_cmd_with_file} --attributes FilePath={run_id}/{filename} RunNumber={run_id}'
```

Comment:
minor: `FilePath` and `RunNumber` attribute keys occur at least twice, maybe it's worth to create constants for them? this will also simplify the potential replacement of values in advance